modules/ROOT/pages/tutorials/private-link.adoc

```diff
@@ -0,0 +1,120 @@
+
+Based on the type of transaction (Read or Write) the driver selects an appropriate node to execute a read or write transaction. 
+The code has the ability to direct the Tx to the appropriate nodes this way.
+
+. Similar to before, the Cloud Virtual Network private DNS is queried and resolves the FQDN to 10.10.10.10. The application's connection is sent to the private link endpoint (10.10.10.10), which forwards the network connection to the private ingress through the private link. 
+The private ingress then directs the connection to the Aura instance with dbid: abcd1234.
+. Finally, the write transaction is received and executed within the Aura instance with dbid: abcd1234.
+
+.Available nodes and their roles
+[cols="1,1"]
+|===
+|abcd1234-core-1.production-orch-0000.neo4j.io
+|role: write
+
+|abcd1234-core-2.production-orch-0000.neo4j.io
+|role: read
+
+|abcd1234-core-3.production-orch-0000.neo4j.io
+|role: read
+|=== 
+
+== Summary
+
+When you create a private link using the Aura console, the private ingress becomes accessible to all projects within the cloud provider’s VPC. 
+
+*The private link maps to an Orchestra not a tenant. 
+We could have multiple orchestras (usually multiple regions)*
+
+For example, if you’re using AWS, you can identify this private ingress by its unique name in your AWS environment.
+
+First, create a private ingress service in Aura VPC, then create a cloud endpoint and try to connect to the service we have provided, then this private link will be accepted.
+It’s all done on the Aura console and you can do it by yourself.
+
+And then there’s a trickier step, but it's the most important thing. 
+*The private DNS.*
+You should configure this private DNS.
+If you are using AWS, then AWS will create this private DNS for you. 
+For Google or Azure, you have to manually create this private DNS.
+Add the Wildcard DNS entry *.production-orch-0000.neo4j.io and point it to the cloud IP endpoint.
+
+
+== Cloud endpoints
+
+A cloud endpoint is essentially a network forwarding rule. 
+It receives network packets, but doesn't open them and forwards them to the Neo4j private ingress via the private link. 
+However, the private ingress only accepts packets from a recognized FQDN.
+
+For example, http://abcd1234-core-1.production-orch-0000.neo4j.io/
+
+There might be some other instances running in the same customer Aura environment. 
+We use this Access Point QDN (Qualified Domain Name) to figure out which Aura instance it is trying to reach.
+*Important: We will not accept the IP address. 
+Only the FQDN will be accepted.*
+
+== Private DNS
+
+Once the private ingress, private link, and private endpoint with the DNS entry are configured, the application will attempt to reach the IP address.
+
+And then it will try to go and check the private link, like “hey, what is the private DNS address for this AFQDN?”
```

Comment:
Yes, maybe it can say:

"Once the private ingress, private link, and private endpoint with the DNS entry are configured, the application will attempt to reach the IP address. Then it will check the private link, to check the private DNS address for the AFQDN.

Then the private DNS will tell the application to go to the IP address of the endpoint." 

Thanks for all the feedback @AlexicaWright 🙌